### DISCUSSION TITLE
Docs domain expired

### DISCUSSION THREAD

**Discussion by TheDevMinerTV** [diced/zipline#447] [Questions] [answered]
It seems like the docs/homepage domain for Zipline expired and got replaced by the ifdetot's filler page.

![image](https://github.com/diced/zipline/assets/29845135/1a0a6aaf-8230-43c2-9b8e-5601827fe294)

**Comment by diced** [comment, marked as answer]
All instances of it were replaced in a recent update so you need to update.

**Reply by TheDevMinerTV** [reply]
Yep, that solved it.
